src/javascript/app_2/Utils/Language/language.js

```diff
@@ -3,7 +3,22 @@ import { get, getAll, urlFor } from '_common/language';
 export const currentLanguage = get();
 
 export const getAllowedLanguages = () => {
-    const exclude_languages = ['ACH'];
+    // TODO Remove all language frome exclude_languages except for ACH when design is ready.
+    const exclude_languages = [
+        'ACH',
+        'DE',
+        'ES',
+        'FR',
+        'ID',
+        'IT',
+        'PL',
+        'PT',
+        'RU',
+        'TH',
+        'VI',
+        'ZH_CN',
+        'ZH_TW',
```

Comment:
instead of adding all here, I suggest just returning 'EN' in language_list, because if API starts sending a new language we have to update this list but if we only return 'EN' as the allowed language no such update is needed

Reply:
much simpler, I agree.